### ISSUE TITLE
Opening a certain ring bricks the entire UI

### ISSUE THREAD

**Issue by filliph** [Tischel/TPie#20]
If I open the following ring:

```
|7JjdbtowFIDvK/EOCO0yygiUie4O6NSx0YqRtOxHu3CTA1hN4so2UDbxZLvYI+0VZjsOMZQIchkpV+Dv2D7++QLR+ffn7+/aRb3eeMM3z9B4X294Ywz2LQkgZPYEx3OrLknDUr3uUKQ6DSiacaBM4wnhiGMSi1DTbmqGArxkgrQ6OzbkELn4l5xCZTXzuhsmovbdMgKKfWY/gM8JbVl1HRhTvEIc7AGhMMKPSWYxwVcxuNPRGUT7m27L5jZJ+xk2fRwHx7Kau9XdzA0ng0VHp3uVggGnoSCcLiFFvZALMkMh2yF3gWevoEfm8xD0YGOB1xStRzhOIxl0IRTHIE62j/ynOSVLtYusj7sga4+QkONntpdMHbVEP5LUeuN5W/8QQgQxZ+J0owjFgW7vH4Vx/wbSIyR960sthDMhmRs9hj6Jh9fShFYG+4QGQLMrOWNtckvJsMN1qWWE5GC603a1T9mlp5GOpRLvoBTtFfyewIwlN6yj3gL7TzEweTFtM5A9LCncJl/S4efe3w0g6gLPuz8dVrfhZPiegUKmrFnv3UM/GRtTST09eDl0XEQ+kUc1W/eIAu9a3abBKwtKZ0HfvS1kwVWOBQavLLDKZkFvUswCp3lcA5NXGpROg8H9qJgGnRwNDF5pYJVNg5E3LaZBK0eD6gWxzC+I04dJMQ3aORoYvPo1sMqmwU3BN0THydHA4JUGpdOgNxoU0+AyRwODVxpYZ2lQEx8/VdfGl6X4S+mpItIwDuBFFiiSyEHJ5FSx5PKsUpyzX4kzmrI0YjSnSVO2dBVsTGGFYS3qRwelLCaXv5JLM+hHxIbxSlhL6CYtdqlw7WL7HwAA//8DAA==
```

It is completely impossible to interact with the "Ring Settings" screen. Nothing can be clicked, not even the close button. I have to create a new empty ring, click the Edit button, then click the Close button in order to actually get that UI off my screen.

If it matters, my native resolution is 1440p, with Windows DPI scaling set to 125% and XIV's scaling set to 150%.

_EDIT:_ I'm now convinced this is due to the preview window. If I drag the Edit Ring pop-up, which I can access by clicking the very bottom left corner, I'm able to return the UI to a working state.

Fresh "Edit" click, broken:
![image](https://user-images.githubusercontent.com/9424286/148479818-f1eb7bf8-2925-4feb-94d0-0deec8356e8e.png)

Preview hidden, working:
![image](https://user-images.githubusercontent.com/9424286/148479864-cc490675-93e8-4dca-87a6-107b352c54b4.png)

EDIT 2:
By observing the change in my mouse cursor from the XIV one to the Windows native one, I've determined that a ring with radius 250 has this big of a preview area:
![image](https://user-images.githubusercontent.com/9424286/148480319-b0342885-1540-4d4b-8b8b-9a603c4b3341.png)

Anything inside that area cannot be interacted with in any way, shape or form.

**Comment by Baelrayne** [comment]
Having the same issue.

Marking the spot where the non-interactivity ends for me.

![image](https://user-images.githubusercontent.com/24845473/148653472-41405ca1-fe25-4adc-be2d-35ac16a2e331.png)

**Comment by Tischel** [comment]
It was an issue with toggle mode.
Fixed in v1.2.0.3
